extensions/typescript-language-features/web/webServer.ts

```diff
@@ -316,7 +360,7 @@ function createServerHost(extensionUri: URI, logger: ts.server.Logger, apiClient
 			}
 
 			try {
-				fs.delete(toResource(path));
+				fs.delete(toResource(path), { recursive: true });
```

Comment:
Why use `recursive` here?

Reply:
probably not appropriate, tbh. Removing.